rdrf/rdrf/features/steps.py

```diff
@@ -123,8 +126,8 @@ def try_to_register(step, registry):
     world.browser.find_element_by_id('registration-submit').click()
 
 
-@step('I should have successfully registered')
-def registration_successful(step):
+@step('I should have successfully registered and would see a "([^"]+)" message')
+def registration_successful(step, success_message):
     # Ensure that the registration has successfully completed
```

Comment:
This is not quite right as sucess_message is overwritten

Better to pass in expected sucess message

get the actual_message from the h3 tag and assert both are the same